core/src/test/java/org/apache/sedona/core/serde/SedonaWKBKryoRegistratorTest.java

```diff
@@ -0,0 +1,56 @@
+package org.apache.sedona.core.serde;
+
+import com.esotericsoftware.kryo.Kryo;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.apache.sedona.core.TestBase;
+import org.apache.sedona.core.serde.WKB.WKBGeometrySerde;
+import org.apache.sedona.core.serde.spatialindex.SpatialIndexSerde;
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.serializer.KryoSerializer;
+import org.junit.*;
+import org.locationtech.jts.geom.Point;
+
+import static org.junit.Assert.assertEquals;
+
+@Ignore
+public class SedonaWKBKryoRegistratorTest extends TestBase {
+
+    public static JavaSparkContext sc;
+    public static SparkConf conf;
+
+    @BeforeClass
+    public static void onceExecutedBeforeAll()
+    {
+        conf = new SparkConf()
+                .setAppName(SedonaWKBKryoRegistratorTest.class.getName())
+                .setMaster("local[2]")
```

Comment:
Fix this as well.

Reply:
Same as [above](https://github.com/apache/incubator-sedona/pull/516#discussion_r615296944)